{"name": "test_case05", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_fixture.Testfixture object at 0x00000217A6021278>, login = None\n\n    def test_case05(self,login):\n        print('=========test_case05 ======')\n>       assert False\nE       assert False\n\ntest_fixture.py:15: AssertionError"}, "start": 1647685262217, "stop": 1647685262217, "uuid": "82c205dd-0137-41c0-842c-8f5849a7d394", "historyId": "c7fc43314746d94842da52b3e53a3462", "testCaseId": "f96d3b96971f1421772f123b46c3d05a", "fullName": "test_fixture.Testfixture#test_case05", "labels": [{"name": "suite", "value": "test_fixture"}, {"name": "subSuite", "value": "Testfixture"}, {"name": "host", "value": "WIN-N5AGDL06G55"}, {"name": "thread", "value": "41080-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_fixture"}]}